tests/test_scuba.py

```diff
@@ -1,18 +1,23 @@
 import pytest
 import shlex
+from typing import Any
 from .utils import assert_vol
 
 from scuba.config import ScubaConfig, ConfigError, OverrideStr
 from scuba.scuba import ScubaContext
 
 
+def make_config(**data: Any) -> ScubaConfig:
```

Comment:
The reasoning for using this utility function instead of invoking `ScubaConfig` directly in each test is explained somewhat in the commit message, but I think a comment here would be a good addition as well. Especially because in its current form it's just a passthrough and seems to provide little value.